Bottom: a1b0b1ab7c6e708d1b95950a54c34c6260b4a4b7
Top:    9f63ea5f723815ad8d2494a2fc34d8250772547b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-12 01:45:56 +0100

Refresh of play-with-glsl-shader-for-rend

---

diff --git a/src/hid/common/hidgl.h b/src/hid/common/hidgl.h
index b83e45b..31d28e8 100644
--- a/src/hid/common/hidgl.h
+++ b/src/hid/common/hidgl.h
@@ -48,7 +48,7 @@ void hidgl_flush_triangles (triangle_buffer *buffer);
 void hidgl_ensure_vertex_space (triangle_buffer *buffer, int count);
 void hidgl_ensure_triangle_space (triangle_buffer *buffer, int count);
 
-inline void
+static inline void
 hidgl_add_vertex_3D_tex (triangle_buffer *buffer,
                          GLfloat x, GLfloat y, GLfloat z,
                          GLfloat s, GLfloat t)
@@ -61,7 +61,7 @@ hidgl_add_vertex_3D_tex (triangle_buffer *buffer,
   buffer->vertex_count++;
 }
 
-inline void
+static inline void
 hidgl_add_vertex_tex (triangle_buffer *buffer,
                       GLfloat x, GLfloat y,
                       GLfloat s, GLfloat t)
